build.gradle

```diff
@@ -120,6 +120,7 @@ subprojects {
 		springOtelVersion = '1.0.0-M8'
 		cloudEventsCoreVersion = '2.4.0'
 		cloudFunctionsFrameworkApiVersion = '1.0.4'
+		opencensusShim = '1.23.1'
```

Comment:
For consistency, this variable should be called `opencensusShimVersion`. Also, the value should include `-alpha`.